src/core/Downloader.ts

```diff
@@ -147,7 +177,9 @@ export class Downloader {
                             if (zip) {
                                 archive.append(buffer, { name: `${item.id}.mp4` });
                             } else {
-                                await fromCallback(cback => writeFile(`${saveDestination}/${item.id}.mp4`, buffer, cback));
+                              var ciFE = readdirSync(`${saveDestination}`).filter(fn => fn.includes(`${item.id}`))
```

Comment:
please avoid using var and sync dependencies

Reply:
Ok, will modify the code to avoid async